WordPress/Classes/Models/WPAccount.m

```diff
@@ -1,9 +1,6 @@
 #import "WPAccount.h"
 #import "WordPress-Swift.h"
 
-static NSString * const WordPressComOAuthKeychainServiceName = @"public-api.wordpress.com";
-static NSString * const JetpackComOAuthKeychainServiceName = @"jetpack.public-api.wordpress.com";
-
```

Comment:
This change is unrelated to this PR. These are unused constants that should've been removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20472.